Frameworks/CoreText/CTTypesetter.mm

```diff
@@ -104,10 +106,15 @@ CFIndex CTTypesetterSuggestLineBreak(CTTypesetterRef typesetter, CFIndex startIn
 */
 CFIndex CTTypesetterSuggestLineBreakWithOffset(CTTypesetterRef ts, CFIndex index, double width, double offset) {
     _CTTypesetter* typesetter = static_cast<_CTTypesetter*>(ts);
-    _CTFrame* frame = _DWriteGetFrame(static_cast<CFAttributedStringRef>(typesetter->_attributedString.get()),
-                                      CFRangeMake(index, [typesetter->_string length] - index),
-                                      CGRectMake(offset, 0, width, FLT_MAX));
-    return ([frame->_lines count] > 0) ? static_cast<_CTLine*>([frame->_lines firstObject])->_strRange.length : 0;
+    CTFrameRef frame = _DWriteGetFrame(static_cast<CFAttributedStringRef>(typesetter->_attributedString.get()),
+                                       CFRangeMake(index, [typesetter->_string length] - index),
```

Comment:
These are  being moved with their associated class being RuntimeBase.
Now i can go a huge change that does all this, but it makes it easier for bugs to go unnoticed.

I'm just breaking out the main classes and it follows through.